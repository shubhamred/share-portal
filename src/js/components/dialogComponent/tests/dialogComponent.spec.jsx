// @flow
import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import DialogComponent from '../dialogComponent';

configure({ adapter: new Adapter() });

describe('Components/DialogComponent', () => {
  it('Renders default DialogComponent', () => {
    const props = {
      children: <p>Test</p>
    };
    const component = shallow(
      <DialogComponent {...props} />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
