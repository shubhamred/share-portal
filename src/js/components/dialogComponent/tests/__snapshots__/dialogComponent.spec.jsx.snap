// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/DialogComponent Renders default DialogComponent 1`] = `
<WithStyles(ForwardRef(Dialog))
  classes={
    Object {
      "paper": "makeStyles-root-1 makeStyles-root-3",
    }
  }
  disableAutoFocus={false}
  disableEnforceFocus={false}
  maxWidth={false}
  open={true}
>
  <WithStyles(ForwardRef(Grid))
    className="grid"
  >
    <div
      className="modalTitle"
      style={Object {}}
    />
    <div
      className="closeBtnContainer"
    >
      <WithStyles(ForwardRef(IconButton))
        className="iconButtonStyle"
        disableRipple={true}
        onClick={[Function]}
      >
        <img
          alt="close"
          height={14}
          src="/assets/close.svg"
          width={14}
        />
      </WithStyles(ForwardRef(IconButton))>
    </div>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(DialogContent))
    className="wrapper"
    style={Object {}}
  >
    <p>
      Test
    </p>
  </WithStyles(ForwardRef(DialogContent))>
</WithStyles(ForwardRef(Dialog))>
`;
